package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sudosos-back-end",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
Please remove package-lock.json from this PR, as package.json itself has not changed.

Is this lock file bump caused by Node 20 btw? Because I only ever see lockfile version 2.